foxpuppet/windows/__init__.py

```diff
@@ -3,5 +3,5 @@
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from foxpuppet.windows.base import BaseWindow  # noqa: F401
-from foxpuppet.windows.browser import BrowserWindow  # noqa: F401
+from foxpuppet.windows.browser.window import BrowserWindow  # noqa: F401
 from foxpuppet.windows.manager import WindowManager  # noqa: F401
```

Comment:
I thought to remember that you decided against the usage of `#noqa` and to skip this warning via the linter settings.